src/ufo-reduce-task.c

```diff
@@ -27,6 +28,28 @@
 
 #include "ufo-reduce-task.h"
 
+typedef enum {
+    MODE_SUM,
+    MODE_MEAN,
+    MODE_MIN,
+    MODE_MAX,
+} Mode;
+
+static GEnumValue mode_values[] = {
+    { MODE_SUM,  "MODE_SUM",  "sum" },
+    { MODE_MEAN, "MODE_SUM", "mean" },
```

Comment:
Will I go straight to hell for this? It makes setup cleaner...

Reply:
No, why? I was doing something similar in the flatten thing AFAIK.